index.html

```diff
@@ -36,19 +37,220 @@ <h2 class="supportive-text">
                                 <a href="https://www.linkedin.com/in/rida-bensalem-7a3329158/"> <img src="Images/Linkedin .svg" alt="linked-in icon"/> </a>
                               </li>
                               <li>
-                                <a href="#"><img src="Images/M.svg" alt="M icon"/> </a>
+                                <a href="#"> <img src="Images/M.svg" alt="M icon"/> </a>
                               </li>
                             <li>
                               <a href="https://github.com/ridabensalem"> <img src="Images/github.svg" alt="github icon" /> </a>
                             </li>
                             <li>
-                                <a href="#"><img src="Images/thumb.svg" alt="thumb icon" /> </a>
+                                <a href="#"> <img src="Images/thumb.svg" alt="thumb icon" /> </a>
                               </li>
                           </ul>
+                   
+                   
                 </div>
             </section>
+            </div>
+            <section class="works-container">
+                <!-- ----------------------------Card number 1--------------------------------------- -->
+              <div class="card-works">
+                    <img src="Images/Snapshoot Portfolio.png" alt="Snapshoot images"> 
+                      <div class="primary-text">
+                        <h3>Tonic</h3>
+                        <div class="description-tags">
+                            <span class="first-tag">CANOPY</span>
+                            <img src="Images/Counter-1.png" alt="counter icon" class="counter">
+                            <span class="second-tag">Back End Dev</span>
+                            <img src="Images/Counter-1.png" alt="counter icon">
+                            <span class="third-tag">2015</span>
+                            </div>
+                            <div class="details">
+                                <p>
+                                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                                </p>
+                        </div>
+                          <div class="tags">
+                            <ul>
+                              <li> <a href="#">html</a> </li>
+                              <li> <a href="#">css</a></li>
+                              <li>  <a href="#">javascript</a> </li>
+                            </ul>
+                          </div>
+                          <div class="button-action">
+                            <button>See Project</button>
+                          </div>
+                      </div>
+                    </div>
+                <!-- -----------------------------------------------Card number 2---------------------------------------- -->
+                                                                         
+                  
+                  <div class="card-works">
```

Comment:
[OPTIONAL] I suggest you use the semantic tags `<article>`. A `div` tag is pretty general. This cards represent standalone parts of the application and they are articles of the section. It makes your code better and friendlier to screen readers